src/Tribe/Repositories/Event.php

```diff
@@ -1526,4 +1444,205 @@ protected function format_item( $id ) {
 
 		return $formatted;
 	}
+
+
+	/**
+	 * Handles the `order_by` clauses for events
+	 *
+	 * @since TBD
+	 *
+	 * @param string $order_by The key used to order events; e.g. `event_date` to order events by start date.
+	 */
+	public function handle_order_by( $order_by ) {
+		/** @var \wpdb $wpdb */
+		global $wpdb;
+
+		$check_orderby = $order_by;
+
+		if ( ! is_array( $check_orderby ) ) {
+			$check_orderby = explode( ' ', $check_orderby );
+		}
+
+		$timestamp_key = 'TIMESTAMP(mt1.meta_value)';
+
+		$by_event_start_date     = isset( $check_orderby['event_date'] )
```

Comment:
For the date related ones do we need to check for all the other possible ways `event_date` is represented? Like when you use `$orm->by( 'date_overlaps', $start, $end );`. I am not sure if it's handled but wanted to make sure.

Reply:
It's handled; filtering and ordering are different matters but the code that this PR shows as changed is really just moved around, not changed in functionality.